Applications/Games/Space Engineers/Steam/script.js

```diff
@@ -1,58 +1,39 @@
+const { getLatestDevelopmentVersion } = include("engines.wine.engine.versions");
 
 new SteamScript()
     .name("Space Engineers")
     .editor("Keen Software House")
     .author("Zemogiter")
-    .appId("244850")
-    .wineVersion("4.14")
+    .appId(244850)
+    .wineVersion(getLatestDevelopmentVersion)
     .wineDistribution("upstream")
     .wineArchitecture("amd64")
     .preInstall((wine) => {
-        new DotNET472(wine).go();
-        new Vcrun2017(wine).go();
+        const wizard = wine.wizard();
+        new dotnet472(wine).go();
+        new vcrun2017(wine).go();
         new DXVK(wine).go();
-        new FAudio(wine).go();
-
+        new xact(wine).go();
         new OverrideDLL(wine)
-            .withMode("native, builtin", [
-                "msvcr120",
-                "xaudio2_0",
-                "xaudio2_1",
-                "xaudio2_2",
-                "xaudio2_3",
-                "xaudio2_4",
-                "xaudio2_5",
-                "xaudio2_6",
-                "xaudio2_7",
-                "xaudio2_8",
-                "xaudio2_9",
-                "x3daudio1_3",
-                "x3daudio1_4",
-                "x3daudio1_5",
-                "x3daudio1_6",
-                "x3daudio1_7"
-            ])
+            .withMode("native, builtin", ["msvcr120"])
+            .withMode("disabled", ["nvapi", "nvapi64"])
             .go();
-
-        const wizard = wine.wizard();
-
-        wizard.message(
-            tr(
-                "You have to install libjpeg62 and libjpeg62-dev or else the thumbnails in New Game menu will be dispalyed as magenta rectangles."
-            )
-        );
-        wizard.message(
-            tr(
-                "Due to JIT compiler issues and the way this game uses multithreading, there are audio stutters. This script will attempt to minimize them but you might also have to enter the alsoft-conf command in terminal and set sample depth to 32bit float and period size to 2048."
-            )
-        );
+        const dxvkConfigFile = wine.prefixDirectory() + "/drive_c/dxvk.conf";
+        touch(dxvkConfigFile);
+        writeToFile(dxvkConfigFile, "dxgi.nvapiHack = False");
+        wizard.message(tr("You may have to install libjpeg62 package if you see thumbnails in New Game menu dispalyed as magenta rectangles."));
+        wizard.message(tr("Due to JIT compiler issues and the way this game uses multithreating, there are audio stutters. If you are on a Debian-based distribution you can try the libFAudio package from this PPA:\nhttps://launchpad.net/~cybermax-dexter/+archive/ubuntu/sdl2-backport"));
+    })
+    .executable("Steam.exe", ["-silent", "-applaunch", "244850", "-no-cef-sandbox", "-skipintro"])
+    .environment((wine) => {
+        const dxvkConfigFile = wine.prefixDirectory() + "/drive_c/dxvk.conf";
+    
+        return '{"DXVK_CONFIG_FILE": "dxvkConfigFile", "STAGING_SHARED_MEMORY": "0", "DXVK_HUD": "compiler", "PULSE_LATENCY_MSEC": "60", "WINEESYNC": "1"}';
```

Comment:
You need to convert the variable into text, this will write as the path "dxvkConfigFile" text.
You can print the whole key to console before returning in order to debug.

Reply:
I did put print (dxvkConfigFile) between const and return and got nothing in the log.

Reply:
Is the HUD working for DXVK (so that we know if the functions is actually executed) ?

Reply:
Negative. No DXVK hud visible.

Reply:
Then you should investigate why it does not work. Try to see if the function is actualy executed or not in `.go()`.

Reply:
I've moved `this._environment = this._environmentFunc(wine);` right below preInstall and added a print debug between these two objects. I got `environment: undefined`